### ISSUE TITLE
a11y addon should highlight violations tab by default

### ISSUE THREAD

**Issue by domyen** [storybooks/storybook#5595]
**Describe the bug**
a11y add-on should highlight “violations” tab when it loads. It currently shows the content for this tab but does not highlight the tab until you click on it.

**To Reproduce**
1. Go to a11y addon
2. See that the violations content is shown
3. But the violations tab is not highlighted (in blue)

**Expected behavior**
The violations tab should be highlighted

**Screenshots**
_Bad_
![image](https://user-images.githubusercontent.com/263385/52826712-004d4a80-3090-11e9-822c-0a764ee35c2c.png)

_Good_
![image](https://user-images.githubusercontent.com/263385/52826721-093e1c00-3090-11e9-9201-e3bb0be6d52f.png)

**Comment by shilman** [comment]
Boo-yah!! I just released https://github.com/storybooks/storybook/releases/tag/v5.0.0-rc.2 containing PR #5646 that references this issue. Upgrade today to try it out!

Because it's a pre-release you can find it on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.
